app/routes/users/components/UserProfile.js

```diff
@@ -363,6 +364,15 @@ export default class UserProfile extends Component<Props, UpcomingEventsProps> {
             ) : (
               <LoadingIndicator loading />
             )}
+            <h3>Dine tidligere arrangementer ({previousEvents.length})</h3>
+            {loading ? (
+              <LoadingIndicator margin={'20px auto'} loading />
+            ) : (
+              <ListEvents
+                events={previousEvents.filter(e => e.userReg.pool !== null)}
```

Comment:
Continuing the discussion in https://github.com/webkom/lego/pull/1553

You could also filter on `userReg.presence` to see if the user actually showed up for the event. However, that depends on whether or not the event used Abacard (and penalties, indirectly). I'm not sure if we can guarantee that right now, so might be better to just show all events for now 😄

Reply:
Yee. If you were registered and attendance isn't set, you get the message `Oppmøte ble ikke sjekket`.
![Screenshot from 2019-04-03 17-43-57](https://user-images.githubusercontent.com/1467188/55492788-190dc180-5638-11e9-97f2-39415289be8b.png)

Reply:
In the spirit of keeping things simple: You could simply exclude registrations where the user is marked `not attending`. That would at least remove events where we know for a fact that the user didn't show up, without having to guess if Abacard was used or not 🙃